languages/swift/exercises/concept/functions/Sources/functions/functions.swift

```diff
@@ -0,0 +1,9 @@
+// Implement function remainingMinutesInOven
+
+// Implement function preparationTimeInMinutes
+
+// Implement function quantities
+
+// Implement function toOz
+
+// Implement function redWine
```

Comment:
For consistency with the `basics` exercise:

```suggestion
// TODO: define the 'remainingMinutesInOven' function

// TODO: define the 'preparationTimeInMinutes' function

// TODO: define the 'quantities' function

// TODO: define the 'toOz' function

// TODO: define the 'redWine' function
```

Note that usually we define stubs for the functions to implement, but as this exercise is about functions themselves, I think TODO's work very well here.